### PR TITLE
docs: add cache status badges

### PR DESCRIPTION
**Pull request by ThetaSinner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+[![Holonix cache trigger](https://github.com/holochain/holonix/actions/workflows/holonix-cache-trigger.yaml/badge.svg)](https://github.com/holochain/holonix/actions/workflows/holonix-cache-trigger.yaml)
+
+[![Holonix cache](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml/badge.svg)](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml)
+[![Holonix cache](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml/badge.svg?branch=main-0.4)](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml)
```
Comment (jost-s) [inline README.md]:
```suggestion
[![Holonix cache 0.4](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml/badge.svg?branch=main-0.4)](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml)
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+[![Holonix cache](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml/badge.svg?branch=main-0.3)](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml)
```
Comment (jost-s) [inline README.md]:
```suggestion
[![Holonix cache 0.3](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml/badge.svg?branch=main-0.3)](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml)
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+[![Holonix cache trigger](https://github.com/holochain/holonix/actions/workflows/holonix-cache-trigger.yaml/badge.svg)](https://github.com/holochain/holonix/actions/workflows/holonix-cache-trigger.yaml)
+
+[![Holonix cache](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml/badge.svg)](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml)
```
Comment (jost-s) [inline README.md]:
```suggestion
[![Holonix cache main](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml/badge.svg)](https://github.com/holochain/holonix/actions/workflows/holonix-cache.yaml)
```

**Review by jost-s — CHANGES_REQUESTED** [review]
Good stuff!
Looking at the badges, I suggest to add the versions.
Screenshot 2024-11-06 at 06 24 58

**Comment by ThetaSinner** [comment]
Yeah, I agree that the names suck. I can't figure out how to make this clear though. The build status is pulled from a given branch but the name isn't actually using the text. It's getting it from the name of the workflow. So I was like okay cool, I'll update the name to `Holonix cache 0.4` and so on on the branches. But then what it actually does is pull the latest build and use the workflow name from that. So you just get `Holonix cache 0.3` for all the names...

I can do some more looking but I'm pretty sure I've run into this before and not found a solution

**Comment by ThetaSinner** [comment]
Not so much by the look of it! https://github.com/orgs/community/discussions/63964

**Comment by jost-s** [comment]
Uh oh, okay. Yeah, lame.
Yes, looking at the client JS repo, the text says "Integration Tests", but the badge reads "Integration Test" which is the name of the workflow. Beyond silly.

Okay, a workaround could be to just write the branch names before the badges.

**Comment by ThetaSinner** [comment]
Yep, that works. The alignment is a bit odd but at least the information is there that way

**Review by jost-s — APPROVED** [review]
Great! There's a design beast in me rearing its head at that alignment. It must shut up.

**Pull request merged**